libucontext.yaml

```
@@ -0,0 +1,43 @@
+package:
+  name: libucontext
+  version: 1.2
+  epoch: 0
+  description: "ucontext function implementations"
+  copyright:
+    - license: "ISC"
+
+environment:
+  contents:
+    packages:
+      - wolfi-base
+      - cmake
+      - build-base
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/kaniini/libucontext
+      tag: v${{package.version}}
+      expected-commit: 4dde3417b4bb4b1b1545bd913be337680b5e28c3
+
+  - runs: |
+      make
+      make DESTDIR="${{targets.destdir}}" install
+
+  - uses: strip
+
+subpackages:
+  - name: "libucontext-dev"
+    description: "ucontext function implementations (development files)"
+    pipeline:
+      - uses: split/dev
+    dependencies:
+      provides:
+        - libucontext-dev=${{package.full-version}}
+
+update:
+  enabled: true
+  github:
+    identifier: kaniini/libucontext
+    strip-prefix: v
+    tag-filter: v
```

```suggestion
    use-tag: true
```
sorry - we do still need use-tags on this package though as https://github.com/kaniini/libucontext/releases only uses tags.  Releases are preferred but only if the upstream project uses github releases.